CHANGELOG.md

```diff
@@ -13,7 +15,6 @@
 
 - **Feature:** Added implementation for orientation lock
 - **Feature:** Added implementation to retrieve Access token details
----
```

Comment:
We don't need separators between each releases?

Reply:
None of the other releases had separator lines, but I guess we could add this to all of them. I'll update this.